editoast/src/generated_data/mod.rs

```diff
@@ -68,12 +69,11 @@ pub trait GeneratedData {
     }
 
     async fn refresh_pool(
-        pool: Arc<DbConnectionPool>,
+        pool: Arc<DbConnectionPoolV2>,
         infra: i64,
         infra_cache: &InfraCache,
     ) -> Result<()> {
-        let mut conn = pool.get().await?;
-        Self::refresh(&mut conn, infra, infra_cache).await
+        Self::refresh(pool.get().await?.deref_mut(), infra, infra_cache).await
```

Comment:
Better to get rid of as much `_pool` variants as possible. Ideally all functions that use a single connection would take a connection as argument.

Reply:
As we discussed, we will leave it as it is.